src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -629,6 +630,15 @@ public class SimpleEvents {
 					"\t\tsend \"The trade was somehow denied!\" to player")
 				.since("INSERT VERSION");
 		}
+		if (Skript.classExists("com.destroystokyo.paper.event.entity.EntityJumpEvent")) {
+			Skript.registerEvent("Entity Jump", SimpleEvent.class, EntityJumpEvent.class, "entity jump[ing]")
+				.description("Called when any entity has jumped.")
```

Comment:
```suggestion
				.description("Called when an entity jumps.")
```

Reply:
Thanks for the suggestion ;) I have pushed the commit.